src/internal/tester/its/m365.go

```diff
@@ -0,0 +1,209 @@
+package its
```

Comment:
could you please use a more descriptive package name? Maybe something like `service` or `scaffold` or something. `its` forces people to read through the code in the package because it lacks any meaning on its own

Reply:
not a fan of `service` or `scaffold`, personally (imo they also lack descriptivity).  Otherwise, sure, I'll figure out a non-abbreviated name by the end.  Also going to punt til later to minimize refactoring.